app/experimenter/kinto/client.py

```diff
@@ -15,6 +15,15 @@ def __init__(self, collection, review=True):
             auth=(settings.KINTO_USER, settings.KINTO_PASS),
         )
         self.review = review
+        self.collection_data = None
+
+    def _fetch_collection_data(self):
+        self.collection_data = (
+            self.collection_data
+            or self.kinto_http_client.get_collection(
+                id=self.collection, bucket=settings.KINTO_BUCKET_WORKSPACE
+            )["data"]
+        )
```

Comment:
Oh yeah you can do it this way or just call it right in __init__ 🤷‍♀️ either works

I guess doing it this way prevents calling it for cases where it's not needed.  Seems good!

Reply:
Also a very unlikely case here is if the data is `{}` it'll always refetch but I don't think that's likely, not worth worrying about for now.

Reply:
Yeah I noticed a handful of spots where we init a KintoClient but never need the collection data, so if the goal is to reduce API calls this seems 👌

Reply:
Yep perfect very nice 👍